### ISSUE TITLE
Short link

### ISSUE THREAD

**Issue by Serovii** [Poniverse/Pony.fm#132]
Hello!
How did you make a short link?

**Comment by TripyVOX** [comment]
What..? Can you rephrase your question @Serovii ?

**Comment by Serkan-devel** [comment]
url-shortening?

**Comment by Serovii** [comment]
Yes

**Comment by AWalGH** [comment]
There's an unwritten secret to get a shorter url than one with the full title. All you have to do is get the full url and take just some important parts out. No idea how long this will continue to work, but it currently does just fine.

Example:
https://pony.fm/tracks/19417-eurobeat-brony-discord-the-living-tombstones-remix

To get a shorter url, you'll change the "tracks/" to just "t"...
https://pony.fm/t

...and then get rid of the text, so that only the numbers remain from the last part of the url...
19417

...You'll then combine those, and you'll get a url that is quite a bit shorter, that will still bring you to the right place.

https://pony.fm/t19417

**Comment by jack126guy** [comment]
It's also available when you click "Share or Embed" on the track page
